src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java

```diff
@@ -14,12 +14,6 @@
 
 import org.testng.annotations.Test;
 
-/**
- * @author Karol 'kkarolk' Kujawiak 1. Create wiki as logged out user 2. Try to create wiki with
- *         wrong user password 3. Try to create wiki with blank user password 4. Try to create wiki
- *         with blank user name 5. Try to create wiki with blank user name 6. Try to create wiki
- *         with invalid user name
- */
 @Test(groups = {"CNW_Anon"})
 public class CreateWikiTests_loggedOutUser extends NewTestTemplate {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00101)